### PR TITLE
Update faq_vac status

### LINKED ISSUE #3112: Add FAQ entry reg. vaccination protection for children under 12
## Enhancement description

Please add a FAQ entry explaining why children below the age of 12 with two vaccinations don't count as fully vaccinated anymore, as of 01.10.22.
The STIKO only recommends 2 vaccinations for healthy children, so many parents believe it is a mistake that the app shows that children under 12 also need a booster to continue to count as fully vaccinated. See e.g. this GitHub Issue: https://github.com/corona-warn-app/cwa-wishlist/issues/862

## Problem and motivation
Make it clearer for the parents to understand why their children need a booster.

---
Internal Tracking ID: [EXPOSUREAPP-14041](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14041)
Internal Tracking ID: [EXPOSUREAPP-14164](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14164)

### PR DESCRIPTION
**Pull request by brianebeling**
Fixes #3112 

Adds new bullet points and extends the main text.

DE:
image

EN:
image

Tests are successful. 

---
Internal Tracking ID: [EXPOSUREAPP-14164](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14164)

### CODE REVIEW

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
One suggestion. Please add non breaking spaces to German abbreviations.

File: src/data/faq_de.json [hunk 1]
```diff
@@ -2538,9 +2538,9 @@
...
...auch für Kinder unter 12 Jahren. Beschränkungen, die an den Nachweis eines vollständigen Impfschutzes knüpfen, bestehen nach Bundesrecht aber derzeit nicht. Daher gibt es auch für Kinder unter 12 Jahren keine Beschränkungen. Es ist möglich, dass in Ihrem Bundesland Schutzmaßnahmen beschlossen werden, von denen Ausnahmen möglich sind, die an den Impfstatus anknüpfen, z.B. Befreiung von der Maskenpflicht bei Vorliegen eines Nachweises über die vollständige Impfung. Bitte beachten Sie dann, dass die Befreiung nicht gilt, wenn Sie nicht vollständig geimpft sind. Das gilt dann auch für Kinder zwischen 6 und 12 Jahren, die in dieser Konstellation weiter Maske tragen müssten. Bitte beachten Sie die Empfehlungen der Ständigen Impfkommission (STIKO) zur COVID-19-Impfung, die auf Grundlage aktueller wissenschaftlicher Kenntnisse nach Alter und Risikofaktoren unterschiedliche Impfabfolgen oder eine unterschiedliche Anzahl von Impfungen empfiehlt. Die aktuellen COVID-19-Impfempfehlungen finden Sie hier: <a href='https://www.rki.de/DE/Content/Infekt/EpidBull/Archiv/2022/Ausgaben/40_22.pdf?__blob=publicationFile' target='_blank' rel='noopener noreferrer'>RKI - Epidemiologisches Bulletin 40</a>",
```
[the remark this answers is not in the document]
Comment (brianebeling) [reply src/data/faq_de.json:2542]: Hi Tim, is it GitHub ignoring the symbols, or is there actually no difference in your suggestion?

[the remark this answers is not in the document]
Comment (Ein-Tim) [reply src/data/faq_de.json:2542]: @brianebeling 

Deems like a GitHub bug. I suggest to change "anknüpfen" to "geknüpft sind".

**Comment by brianebeling** [comment]
Thank you for the suggestions @Ein-Tim! I think something went wrong somewhere, but I included them in the PR.

**Comment by MikeMcC399** [comment]
There should be a space between "§" and the number of the paragraph i.e. `§ 22a`.

**Review by Ein-Tim — APPROVED** [review]
LGTM!

**Comment by MikeMcC399** [comment]
Other FAQ articles use the link https://www.gesetze-im-internet.de/ifsg/__22a.html to refer to IfSG § 22a. This is the official site. 

https://www.buzer.de/ is a very useful site, however it is not owned by the government. See https://www.buzer.de/i.htm

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks again; lets use https://www.gesetze-im-internet.de/ifsg/__22a.html

**Review by GisoSchroederSAP — APPROVED** [review]
Changes are fine, as well as changing the details and links

**Review by dsarkar — APPROVED** [review]

**Pull request merged**